host-go/config/model/lens.go

```diff
@@ -27,4 +27,7 @@ type LensModule struct {
 	//
 	// The lens module must expose a `set_param` function if values are provided here.
 	Arguments map[string]any
+
+	// If true WASI imports will be enabled for this module.
+	EnableWASI bool
```

Comment:
question: This param doesn't make sense to me, but it might be due to the lack of tests.  If this is enabled for wasm-only modules, do they still work?